device/bgp/bgpv6.yaml

```diff
@@ -7,18 +7,17 @@ components:
       description: >-
         Container for BGPv6 peer settings and routes.
       type: object
+      required: [dut_address]
```

Comment:
removal of default should make this a required property

Reply:
Most of tgens would take local address from ipv4 stack and populate the local address. removal of default would make it to None and the tgen default values are applied.

Reply:
@ANISH-GOTTAPU the comment that concrete implementations would take the local address from the ipv4 stack and apply it to the local address is implicit, the model still requires that property to either have a default value or be required.

Reply:
`local_address` is now a required field